contracts/src/lib/SubnetIDHelper.sol

```diff
@@ -42,7 +42,7 @@ library SubnetIDHelper {
     }
 
     function toString(SubnetID calldata subnet) public pure returns (string memory) {
-        string memory route = string(abi.encodePacked("/r", Strings.toString(subnet.root)));
+        string memory route = string.concat("/r",Strings.toString(subnet.root));
```

Comment:
```suggestion
        string memory route = string.concat("/r", Strings.toString(subnet.root));
```

Reply:
Was Aderyn complaining about this, or is it just an opportunistic optimization?

Reply:
Aderyn was in fact complaining about this - here is the latest run https://github.com/consensus-shipyard/ipc/actions/runs/8782737964/job/24097651471 - when this error is fixed, the test file that I changed also shows the same error, and when that second error is fixed there is a 3rd reported error related to the diamond initialize function